sysvabi64/sysvabi64.rst

```diff
@@ -1435,10 +1435,17 @@ HWCAP_ flags.
     unsigned long _size; /* Size of the struct, so it can grow.  */
     unsigned long _hwcap;
     unsigned long _hwcap2;
+    unsigned long _hwcap3;
+    unsigned long _hwcap4;
   };
 
   typedef struct __ifunc_arg_t __ifunc_arg_t;
 
+The second paramter is a pointer to a ``__ifunc_arg_t`` object and its
```

Comment:
(also: mention that this is only true if `_IFUNC_ARG_HWCAP` is set in the _first_ parameter?)